protobuf/lib/src/protobuf/generated_message.dart

```diff
@@ -162,15 +167,18 @@ abstract class GeneratedMessage {
     }
   }
 
+  /// Serialize the message.
```

Comment:
Should we mention the serialization format (i.e. binary protobuf encoding)?

Reply:
@sigurdm PTAL.